Purchases/SwiftSources/LocalReceiptParsing/BasicTypes/ASN1ObjectIdentifier.swift

```diff
@@ -0,0 +1,15 @@
+//
+// Created by Andrés Boedo on 7/29/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+enum ASN1ObjectIdentifier: String {
+    case data = "1.2.840.113549.1.7.1"
+    case signedData = "1.2.840.113549.1.7.2"
+    case envelopedData = "1.2.840.113549.1.7.3"
+    case signedAndEnvelopedData = "1.2.840.113549.1.7.4"
+    case digestedData = "1.2.840.113549.1.7.5"
+    case encryptedData = "1.2.840.113549.1.7.6"
```

Comment:
couldn't find any docs on these, but will try to find something before making the PR final